modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -33,6 +33,89 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
+
+The following tables describe the default settings for API proxies based on different Mule versions:
+
+[[default-setting-mule-3]]
+==== Default Setting by API Proxy Versions for Mule 3.x Applications
+
+[%header%autowidth.spread]
+|===
+| Mule Version | API Proxy | Default Setting | UI Support 
+| 3.7x | RAML or WDSL | Disabled | No
+| 3.8x | RAML or WDSL | Disabled | Yes
+| 3.9x | RAML or WDSL | Disabled | Yes
+|===
+
+Mule 3.x applications and API proxy compatibility synopsis:
+
+* Only regular proxy validations are available for Mule 3.x applications.
+* Validations are performed for only WSDL and RAML proxies.
+* Proxies are not versioned in Mule 3.x applications.
+* You can edit proxies from API Manager for Mule versions 3.8.x and later.
+
+[[default-setting-mule-4]]
+==== Default Setting by API Proxy Versions for Mule 4.x Applications
+
+[%header%autowidth.spread]
+|===
+| Mule Version | API Proxy | Default Setting | UI Support
+| 4.x| RAML 1.1.x            | Enabled  | No
+| 4.x| RAML 1.2.x            | Enabled  | Yes
+| 4.x| REST 2.0.x            | Enabled  | No
+| 4.x| REST 2.1.0 or later  | Enabled   | Yes
+| 4.x| WSDL 1.2.0 and earlier  | Disabled | No
+| 4.x| WSDL 1.2.1 and later  | Disabled | Yes
+|===
+
+Mule 4.x applications and API proxy compatibility synopsis:
+
+* For Mule 4.x applications, both regular and strict validations are available.
+* Validations are performed for only WSDL and RAML or REST proxies.
+* Proxies are versioned in Mule 4.x applications. 
+* You can edit RAML or REST proxies from API Manager for RAML proxy version 1.2.0 and REST proxy version 2.1.0 and later.
+
+
+=== API Specifications for Strict Validations
+When enabling strict validations for Mule 4 applications, you must consider your API architecture. If your Mule application has a proxy or a load balancer in front of it, the web server might modify the client request to include additional headers. In such cases, the Mule application denies the incoming request due to a non-valid header error.
+
+When enabling strict validations for Mule 4 applications, you must consider your API architecture. If your Mule application has a proxy or a load balancer in front of it, the web server might modify the client request to include additional headers. In such cases, the Mule application denies the incoming request due to a non-valid header error. 
+
+Note that the newer versions of CloudHub and RuntimeFabric also include headers when sending requests to the API.
+
+==== Runtime Fabric headers
+
+The following headers are added by Edge in Runtime Fabric:
+[source,RAML,linenums]
+----
+headers:
+  x-correlation-id:
+    type: string
+  x-forwarded-for:
+    type: string
+  via:
+    type: string
+  x-forwarded-host:
+    type: string
+----
+
+Therefore, consider adding these headers in the API specification when using Edge.
+
```

Comment:
Hi @priyaayyar, these last paragraphs are the ones we need to remove.

Reply:
Ok Done!